projects/cashmere/src/lib/util.ts

```diff
@@ -1,3 +1,11 @@
 export function parseBooleanAttribute(value: any): boolean {
-    return value != null && value.toString() !== 'false';
+    if (typeof value === 'boolean') {
+        return value;
+    } else if (value.toLowerCase() === 'false') {
+        return false;
+    } else if (value.toLowerCase() === 'true' || value === '') {
```

Comment:
I'd expect that if it's not explicitly `'true'` or `'false'` that it'd throw an error.

What's the scenario you'd see where we would want `''` to return `true`?

Reply:
I could maybe see that if it wasn't included it could default to `''` and then we'd just default to `false` maybe?

Reply:
This is an interesting one because I originally thought the same as you.  But I realized when I made this update that this was throwing an error:
`<input hcInput placeholder="Enter username" disabled>`
...and it's because the component is using `parseBooleanAttribute` on the "disabled" value.  So it's correct syntax to write that as either `<input disabled>` or `<input disabled=true>` and it shouldn't be throwing an error.  And in the first example where the value isn't included, it's expected that the value is true.
So that's how I arrived where I did - open to suggestions though.

Reply:
Interesting. That makes sense! Thanks for the info.